src/test/java/core/ControllerCompareTest.java

```diff
@@ -628,12 +634,12 @@ public void testMultipleReportFormatsPrintResults() throws IOException {
         reader.close();
 
         assertTrue(jsonResult.isJsonObject());
-        assertEquals("4.0", jsonResult.getAsJsonObject().get("results").getAsJsonObject().get("averagevalue").getAsString());
+        assertEquals("[ 4.0 ]", jsonResult.getAsJsonObject().get("results").getAsJsonObject().get("averagevalue").getAsString());
 
         deleteFiles(RESULT_FILE_NAME + "RawResults", RESULT_FILE_NAME + "JSON");
 
         // Check console
-        assertTrue("Correct result wasn't printed to console.", outContent.toString().contains("4.0"));
+        assertTrue("Correct result wasn't printed to console.", outContent.toString().contains("[ 4.0 ]"));
```

Comment:
These are two of the places where I was saying in another comment these spaces in the array aren't necessary.